stups-spring-oauth2-server/src/main/java/org/zalando/stups/oauth2/spring/server/DefaultTokenInfoRequestExecutor.java

```diff
@@ -74,8 +77,16 @@ public DefaultTokenInfoRequestExecutor(final String tokenInfoEndpointUrl, RestOp
     //@formatter:on
 
     public static RestTemplate buildRestTemplate() {
+        return buildRestTemplate(TokenInfoResponseErrorHandler.getDefault());
+    }
+
+    public static RestTemplate buildRestTemplate(List<HttpStatus> statusList) {
```

Comment:
It would be nice to have either some Java Docs here, or a less general parameter name. e.g. "invalidTokenStatusCodes".